test/unit/test_row_factory.py

```diff
@@ -0,0 +1,54 @@
+from unittest import mock
+import pytest
+from etlhelper.row_factories import namedtuple_rowfactory
+import sqlite3
+
+
+# Test uses sqlite3 cursor, but all dbapi compliant cursors will be the same.
+mock_cursor = mock.MagicMock(sqlite3.Cursor)
```

Comment:
Best to have a new mock per test in case it remembers function calls between tests.